ClusterManagement/Cluster_url_clarification.md

```diff
@@ -0,0 +1,23 @@
+### URL difference
+[Offcial doc](https://hadoop.apache.org/docs/r2.7.3/) lists 
+all the APIs for each hadoop relevant service. However, because of the Gateway policy, HDInsight sometimes uses different URL to get the info. 
+This doc is an URL clarifaction for cases we knew so far. We would keep this file updated once we see more cases.
+
+### MapReduce API
+* Apache Hadoop
+```
+ http://<proxy http address:port>/proxy/{appid}/ws/v1/mapreduce
```

Comment:
How do they get this appid? Is there a way to list the appids?

Reply:
Like, if they submit the job through templeton, oozie or something else, how can get the appid

Reply:
There are other APIs to get the overall app info. such as https://<url>/ws/v1/cluster/apps/. I believe this API is specifically used for single application. The reason i put this one here is because this URL is different from the official doc, while the others are same.